.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,21 @@
 ## Checklist
 
-- [ ] Change(s) are motivated and described in the PR description
-- [ ] Testing strategy is described if automated tests are not included in the PR
-- [ ] Risks are described (performance impact, potential for breakage, maintainability)
-- [ ] Change is maintainable (easy to change, telemetry, documentation)
-- [ ] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html) are followed or label `changelog/no-changelog` is set
-- [ ] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/))
+- [ ] The PR description includes an overview of the change
+- [ ] The PR description articulates the motivation for the change
+- [ ] The change includes tests OR the PR description describes a testing strategy
+- [ ] The PR description notes risks associated with the change, if any
+- [ ] Newly-added code is easy to change
+- [ ] The change follows the [library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html)
+- [ ] The change includes or references documentation updates if necessary
 - [ ] Backport labels are set (if [applicable](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting))
-- [ ] If this PR changes the public interface, I've notified `@DataDog/apm-tees`.
```

Comment:
Should we be communicating this elsewhere? Or maybe update the Slackbot to automatically post in one of the TEE channels with the new release note could be nice

Reply:
Yes, more communication is better in this case.